frontend/src/Components/CardSelector/CardSelector.tsx

```diff
@@ -13,11 +13,17 @@ const SPECIAL_ICONS: { [value in CardValue]?: JSX.Element } = {
   [VOTE_OBSERVER]: <IconObserver />,
   [VOTE_COFFEE]: <IconCoffee />,
 };
-const getCard = (cardValue: CardValue, isSelected: boolean, setVote: WebSocketApi['setVote']) => {
+const getCard = (
+  cardValue: CardValue,
+  isSelected: boolean,
+  setVote: WebSocketApi['setVote'],
+  enabled: boolean
+) => {
   const isObserver = cardValue === VOTE_OBSERVER;
 
   return (
     <button
+      disabled={!enabled}
```

Comment:
We would need to disabled the kick user button as well.

Reply:
Done, and I also extended the unit tests to check the disabled elements